public-ui/src/app/data-browser/views/pm/pm-react.component.tsx

```diff
@@ -0,0 +1,488 @@
+                          selectedConcept.conceptId !== '903111' && selectedConcept.conceptId !== '903120' && selectedConceptUnit ?
+                          <React.Fragment>
+                          {  this.getCountAnalysis(selectedConceptUnit).map((gender, index) => {
+                               const chartKey = gender.stratum3 + '-' + index;
+                               return <div className='bs-chart-item' key={chartKey}>
+                                 <ValueReactChartComponent conceptId={selectedConcept.conceptId}
+                                   valueAnalysis={this.getValueAnalysis()}
+                                   domainCountAnalysis={domainCountAnalysis}
+                                   genderId={gender.stratum3}
+                                   chartTitle={gender.analysisStratumName + ' - ' + (gender.countValue <= 20 ? '&le; ' : '') +
+                                   gender.countValue.toLocaleString()} key={chartKey}></ValueReactChartComponent></div>;
+                               })
+                          }
+                          </React.Fragment>
+                          : selectedConcept.analyses.measurementValueGenderAnalysis ?
+                            <div className='chart-item stacked-chart-item'>
+                              <ValueReactChartComponent conceptId={selectedConcept.conceptId}
+                                valueAnalysis={selectedConcept.analyses.measurementValueGenderAnalysis[0]}
+                                domainCountAnalysis={domainCountAnalysis}
+                                genderId='stacked gender'
+                                chartTitle='stacked chart'>
+                              </ValueReactChartComponent>
+                            </div> : null
+                          : null
+                       }
+                       {  selectedConcept.analyses && selectedConcept.analyses.ageAnalysis ?
+                          <div className='chart-item age-chart'>
+                            <div className='bs-title' style={styles.bsTitle}>Age When Physical Measurement Was Taken
+                              <TooltipReactComponent
+                                tooltipKey='pmAgeChartHelpText'
+                                label='Physical Measurements tooltip hover'
+                                searchTerm='TODO replace search text in here'
+                                action={'Hover on pm age chart of concept ' + selectedConcept.conceptName}>
+                              </TooltipReactComponent>
+                            </div>
+                            <AgeChartReactComponent ageAnalysis={selectedConcept.analyses.ageAnalysis}
+                              ageCountAnalysis={domainCountAnalysis.ageCountAnalysis} domain='pm' selectedResult=''>
+                            </AgeChartReactComponent>
```

Comment:
Use the self-closing syntax:
```suggestion
                              ageCountAnalysis={domainCountAnalysis.ageCountAnalysis} domain='pm' selectedResult=''/>
```

Reply:
Thanks. changed this.